integration_tests/src/main/python/arithmetic_ops_test.py

```diff
@@ -535,3 +535,35 @@ def test_div_by_zero_ansi(expr):
 @pytest.mark.parametrize('expr', ['1/0', 'a/0', 'a/b'])
 def test_div_by_zero_nonansi(expr):
     _test_div_by_zero(ansi_mode='nonAnsi', expr=expr)
+
+
+def _test_div_overflow(ansi_mode, expr):
+    ansi_conf = {'spark.sql.ansi.enabled': ansi_mode == 'ansi'}
+    data_gen = lambda spark: two_col_df(
```

Comment:
If you know what the data is that you want, you don't have to use data_gen to do it. Just put the exact data in that you want.

Reply:
Got it. Updated.